sympy/matrices/matrices.py

```diff
@@ -932,7 +932,7 @@ def columnspace(self, simplify=False):
 
         return [self.col(i) for i in pivots]
 
-    def nullspace(self, simplify=False):
+    def nullspace(self, iszerofunc=_iszero, simplify=False):
```

Comment:
It isn't good practice for backwards compatibility to insert keyword args before existing ones, because you can technically call the function by only relying on argument order. Move `iszerofunc` after `simplify`.